src/obr/core/core.py

```diff
@@ -142,6 +142,32 @@ def logged_func(func, doc, **kwargs):
     doc["history"].append(res)
 
 
+def get_mesh_stats(owner_path: str) -> dict:
+    """Check constant/polyMesh/owner file for mesh properties
+    and return it via a dictionary"""
+    nCells = None
+    nFaces = None
+    if Path(owner_path).exists():
+        with open(owner_path, "r", errors="replace") as fh:
+            read = True
+            FoamFile = False
+            found_note = ""
+            while read:
+                # A little parser for the header part of a foam file
+                # TODO this should be moved to OWLS
+                line = fh.readline()
+                if "FoamFile" in line:
+                    FoamFile = True
+                if FoamFile and line.strip().startswith("}"):
+                    read = False
+                if FoamFile and "note" in line:
+                    found_note = line
```

Comment:
```suggestion
            is_foamfile = False
            found_note = ""
            while read:
                # A little parser for the header part of a foam file
                # TODO this should be moved to OWLS
                line = fh.readline()
                if "FoamFile" in line:
                    is_foamfile = True
                if is_foamfile and line.strip().startswith("}"):
                    read = False
                if is_foamfile and "note" in line:
                    found_note = line
```

Reply:
I am unsure about this change. 
```
                if not "FoamFile" in line:
                    continue
```
The "FoamFile" is something like a keyword which signals the start of a "dictionary". Thus, we set  FoamFile to be true. and only parse between FoamFile and the end of the dictionary signaled by "}".  But if continue if "FoamFile" is not in the current line we miss the dictionary part.

Reply:
Ah yes. You're right. Then i would roll back my suggestion to rename the variable to represent a boolean better:
```suggestion
            is_foamFile = False
            found_note = ""
            while read:
                # A little parser for the header part of a foam file
                # TODO this should be moved to OWLS
                line = fh.readline()
                if "FoamFile" in line:
                    is_foamFile = True
                if is_foamFile and line.strip().startswith("}"):
                    read = False
                if is_foamFile and "note" in line:
                    found_note = line
```